js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -377,7 +409,61 @@ export class OpenAIModel implements PromptCompletionModel {
             return this.returnError(err, input);
         }
     }
+    /**
+     * Converts the messages to ChatCompletionMessageParam[].
+     * @param {Message<string>} messages - The messages from result.output.
+     * @returns {ChatCompletionMessageParam[]} - The converted messages.
+     */
+    private convertMessages(messages: Message<string>[]): ChatCompletionMessageParam[] {
+        const params: ChatCompletionMessageParam[] = [];
+        // Iterate through the messages and check for action calls
+        for (const message of messages) {
+            let param: ChatCompletionMessageParam = {
+                role: 'user',
+                content: message.content ?? ''
+            };
+            const toolCallParams: ChatCompletionMessageToolCall[] = [];
```

Comment:
maybe move this line down to the assistant block since its only referenced there